src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -239,7 +239,8 @@ function Overview({ bucket, ingestionStates }: Props) {
     tags.status === 'success' && tags.value?.[BUCKET_TAG_VEEAM_APPLICATION];
   const isVeeamBucket =
     (veeamTagApplication === VeeamApplicationType.VEEAM_BACKUP_REPLICATION ||
-      veeamTagApplication === VeeamApplicationType.VEEAM_OFFICE_365) &&
+      veeamTagApplication === VeeamApplicationType.VEEAM_OFFICE_365 ||
+      veeamTagApplication === VeeamApplicationType.VEEAM_OFFICE_365_V8) &&
```

Comment:
nit: could be simplified `[VeeamApplicationType.VEEAM_BACKUP_REPLICATION, VeeamApplicationType.VEEAM_OFFICE_365, VeeamApplicationType.VEEAM_OFFICE_365_V8].includes(veeamTagApplication)`. not necessary to do it now, just a dumb suggestion